src/crypto/crypto-ops.h

```diff
@@ -161,5 +163,12 @@ void ge_sub(ge_p1p1 *r, const ge_p3 *p, const ge_cached *q);
 void fe_add(fe h, const fe f, const fe g);
 void fe_tobytes(unsigned char *, const fe);
 void fe_invert(fe out, const fe z);
+void fe_mul(fe out, const fe, const fe);
+void fe_0(fe h);
 
 int ge_p3_is_point_at_infinity_vartime(const ge_p3 *p);
+
+// miscellaneous
```

Comment:
Since this is in an important file, I'd leave a more descriptive comment here, more or less what you put in the post, explaining why this needs to be exposed in this header file.

Reply:
Oh looks like these are leftovers I'm not actually using. Will remove